tst/testbugfix/00078.tst

```diff
@@ -0,0 +1,3 @@
+# 2005/07/13 (JS)
+gap> PerfectGroup(7800,1);; # load perf2.grp
+gap> PerfectGroup(7680,1);; # should load perf1.grp, gives error in 4.4.5
```

Comment:
This tests is redundant now, we have a test that loads the whole perfect groups library.